### ISSUE TITLE
bug: exp module perspective mismatch

### ISSUE THREAD

**Issue by ivokub** [Consensys/corset#157]
Trying to open `zkevm.bin` through the libcorset Go bindings:
```go
corset, err := C.corset_from_file(C.CString("/home/ivo/temp/cur/zkevm.bin"))
```

I get the error:
```
columns exp.TRIM_BYTE%computation and exp.AUX_255_HOOD are in different perspectives
```
from the library.

Corset version: 067b412ef55ac96bc356976b9ac768452ce1589b
Constraints version: f1d213b75a2a226200b8d394dffce9d1b94b5979

**Comment by DavePearce** [comment]
Minimal test case for reproduction:

```lisp
(module test)
(defcolumns (FLAG))
(defperspective tmp FLAG ((BYTE :byte@prove)))
```

Command-line:

```
corset wizard-iop -vv -o test.bin test.lisp 
```

Observe that removing `@prove` and its fine.

**Comment by DavePearce** [comment]
This is now resolved by #182.  I believe there are likely to be issue around perspectives still, which will be addressed in #101.
